packages/table/src/interactions/draggable.tsx

```diff
@@ -10,6 +10,7 @@ import * as PureRender from "pure-render-decorator";
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
+import { Utils } from "../common/utils";
```

Comment:
future issue: this is _super_ awkward usage. why not just export individual functions like we do in other utils files?